integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -639,6 +660,53 @@ public int rmdir(String path) {
     return rmInternal(path);
   }
 
+  /**
+   * Gets the filesystem statistics.
+   *
+   * @param path The FS path of the directory
+   * @param stbuf Statistics of a filesystem
+   * @return 0 on success, a negative value on error
+   */
+  @Override
+  public int statfs(String path, Statvfs stbuf) {
+    LOG.trace("statfs({})", path);
+    ClientContext ctx = ClientContext.create(sConf);
+
+    try (BlockMasterClient blockClient =
+             BlockMasterClient.Factory.create(MasterClientContext.newBuilder(ctx).build())) {
+      Set<BlockMasterInfo.BlockMasterInfoField> blockMasterInfoFilter =
+          new HashSet<>(Arrays.asList(
+              BlockMasterInfo.BlockMasterInfoField.CAPACITY_BYTES,
+              BlockMasterInfo.BlockMasterInfoField.FREE_BYTES,
+              BlockMasterInfo.BlockMasterInfoField.USED_BYTES));
+      BlockMasterInfo blockMasterInfo = blockClient.getBlockMasterInfo(blockMasterInfoFilter);
+
+      // this is only an estimation,
+      // as user may set different block size for different files
+      long blockSize = sConf.getBytes(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT);
```

Comment:
@apc999 Do we truly want to use the `default block size`? If users do not change it, the default block size is 512MB. Even if Alluxio only contains a small file (several bytes), the `df` command will show 512MB used capacity.

Reply:
understand this is confusing. But I don't have other good idea here. Do you have any suggestion?

Reply:
i would prefer to make the block size we show in linux similar to linux block size, like 4KB so that we can show more accurate capacity information. But not sure will it cause other problems because our actual block size is far larger than 4KB.

Reply:
@liuhongtong when you run `df`, will this command show the correct values for used/available space if we use a much larger block size?

Reply:
@apc999 larger block size may result in inaccurate values of used/available space. See my comment below.